Cargo.lock

```diff
@@ -15,9 +15,9 @@ dependencies = [
 
 [[package]]
 name = "aho-corasick"
-version = "1.0.2"
```

Comment:
the Cargo.lock changes are unrelated right? Would have been better to as a separate commit with then.

Reply:
This combines the two lock files we had, so some of the changes are required, I don't remember if I did that with `cargo update` or `cargo generate-lockfile`, but if the output is breaking in any way that is an issue non-locked dependencies in `Cargo.toml`.